src/modules/previewpane/powerpreview/powerpreview.h

```diff
@@ -0,0 +1,48 @@
+#include "pch.h"
+#include <interface/powertoy_module_interface.h>
+#include <common.h>
+#include "trace.h"
+#include "settings.h"
+
+using namespace PowerPreviewSettings;
+
+extern "C" IMAGE_DOS_HEADER __ImageBase;
+
+// Implement the PowerToy Module Interface and all the required methods.
+class PowerPreviewModule : public PowertoyModuleIface
+{
+private:
+    // The PowerToy state.
+    bool m_enabled = false;
+    ExplrSVGSttngs m_explrSVGSettings = ExplrSVGSttngs();
+    PrevPaneSVGRendrSettings m_prevPaneSVGSettings = PrevPaneSVGRendrSettings();
+    PrevPaneMDRendrSettings m_prevPaneMDSettings = PrevPaneMDRendrSettings();
+    std::wstring m_moduleName;
+
+    // Load and save Settings.
+    void init_settings();
+
+public:
+    PowerPreviewModule()
+    {
+        this->m_moduleName = GET_RESOURCE_STRING(IDS_MODULE_NAME);
+
+        this->m_explrSVGSettings = ExplrSVGSttngs();
```

Comment:
nit:  The default constructors will be called for you.   Also its generally preferable to initialize member variables via the initializer list if possible, as opposed to the constructor body.

Reply:
https://github.com/microsoft/PowerToys/pull/1075/commits/f2b766860c3a77391da51d6dcb1c968032318010